### DISCUSSION TITLE
Is psycopg2 2.8.6 compatible with PostgreSQL 14.x?

### DISCUSSION THREAD

**Discussion by luzfcb** [psycopg/psycopg2#1441] [Q&A]
Problem: 

The infrastructure team at the company I work for is starting the upgrade process to `PostgreSQL 14`, I have a few projects that are still running `Django 2.2`. 
It will be a few months before we can migrate all apps and dependencies to be compatible with Django 3.2.

I tried to update `psycopg2` to `2.9.3` however I got the `database connection isn't set to UTC` error.

After some searching, I found that the solution is to fix the version of `psycopg2` to `2.8.6`, however, I couldn't find out in the related issues whether or not `psycopg2 2.8.6` is compatible with `PostgreSQL 14.x`

Questions:
1 - Is `psycopg2 2.8.6` compatible with `PostgreSQL 14.x`?
2 - Would there be any (known) issues when using `psycopg2 2.8.6` with `PostgreSQL 14.x`?

Related: 
https://github.com/django/django/pull/14530
https://github.com/psycopg/psycopg2/issues/1293

@dvarrazzo

**Comment by dvarrazzo** [comment]
Yes

**Reply by HiThereStranger** [reply]
@dvarrazzo This is very helpful info, Thank you! Is `psycopg2 2.8.6` compatible with `PostgreSQL 16.x`? If so are there any known issues with this combo?

**Comment by dvarrazzo** [comment]
> Is psycopg2 2.8.6 compatible with PostgreSQL 16.x?

Yes

**Reply by HiThereStranger** [reply]
@dvarrazzo Thank you for the speedy reply! Much appreciated.
